.circleci/config.yml

```diff
@@ -62,6 +62,33 @@ jobs:
       - run: docker login -u="${DOCKER_USERNAME}" -p="${DOCKER_PASSWORD}"
       - run: make docker-push
 
+  publish_release:
+    docker:
+      # Available from https://hub.docker.com/r/circleci/golang/
+      - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/improbable-eng/thanos
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.07.0-ce
+      - attach_workspace:
+          at: .
+      - run: make promu
+      - run: promu crossbuild tarballs
+      - run: promu checksum .tarballs
+      - run: promu release .tarballs
+      - store_artifacts:
+          path: .tarballs
+          destination: releases
+      - run: ln -s .build/linux-amd64/thanos thanos
+      - run: make docker DOCKER_IMAGE_TAG=$CIRCLE_TAG
+      - run: docker run thanos --help
+      - run: docker login -u="${DOCKER_USERNAME}" -p="${DOCKER_PASSWORD}"
+      - run: |
+          if [[ "$CIRCLE_TAG" =~ ^v[0-9]+(\.[0-9]+){2}$ ]]; then
+            make docker-tag-latest DOCKER_IMAGE_TAG=$CIRCLE_TAG
```

Comment:
Why are we doing this? I think it might be enough to do `make docker-push DOCKER_IMAGE_TAG=$CIRCLE_TAG` and remove `make docker-tag-latest` completely.

Reply:
This updates the `:latest` "symlink" tag in docker hub. The `docker-push` just updates the `vX.X.X` tag.

Reply:
I don't think it updates anything. This just `tags` local docker. Anyway, I think we want to get rid of `latest` totally, because it is super confusing when users claim answers (when asked what Thanos version you use) just "latest" ;p Can we just publish `vX.X.X`?

Reply:
wow. hm. Well maybe that is something to configure, but if you don't explictly push `latest` tag - it won't be updated. See: https://hub.docker.com/r/improbable/thanos/tags/ No latest tag, even though we are pushing on every master commit.